supply-chain.md

```diff
@@ -80,13 +80,14 @@ the problem and helps when developing prototype utilities and queries with GUAC
 
 ## Step 5: Mark packages as bad when a security incident occurs
 
-A new security incident has occurred and various communities have pointed out that a particular package is affected. In this scenario, the package "maven/org.apache.logging.log4j/log4j-core" has a vulnerability (yikes!).
+A new security incident has occurred and various communities have pointed out that a particular package is affected. In this scenario, the package debian package
+"tzdata" has been found to have a critical vulnerability (yikes!). We know the package and the specific version that is vulnerable. Can we be proactive with this information and quickly find where this package is being used?
```

Comment:
@sarahlinnl to review this change

Reply:
Change "the package debian package" to "the debian package". 

I would change the last two sentences to "Now that we know the package and the vulnerable version, can we use this information to quickly find where this package is being used?"